inst/templates/tidy_contributing.md

```diff
@@ -0,0 +1,156 @@
+# Contributing to {{{package}}}
+
+-   [Prerequisites](#prerequisites)
+-   [Package mechanics](#package-mechanics)
+    -   [Fork, clone, branch](#fork-clone-branch)
+    -   [Establish a baseline](#establish-a-baseline)
+    -   [Style](#style)
+    -   [Documentation](#documentation)
+    -   [Testing](#testing)
+    -   [NEWS](#news)
+    -   [Committing your changes](#commiting-your-changes)
+-   [Making the pull request](#making-the-pull-request)
+    -   [Review, revise, repeat](#review-revise-repeat)
+-   [Resources](#resources)
+-   [Code of Conduct](#code-of-conduct)
+
+This explains how to propose a change to {{{package}}} via a pull request using
+Git and Github. 
+
+For more general info about contributing to the tidyverse, see the 
+[Resources](#resources) at the end of this document.
+
+## Prerequisites
+
+Before you do a pull request, you should always file an issue and make sure
+someone from the tidyverse team agrees that it’s a problem, and is happy with
+your basic proposal for fixing it. If you’ve found a bug, first create a minimal
+[reprex](https://www.tidyverse.org/help/#reprex).
+
+## Package mechanics
+
+### Fork, clone, branch
+
+The first thing you'll need to do is to fork the 
+[{{{package}}} GitHub repo](https://github.com/tidyverse/{{{package}}}), and 
+then clone it locally. We recommend that you create a branch for each PR.
+
+### Establish a baseline
+
+Before changing anything, make sure the package still passes `R CMD check`
+locally for you.
+
+``` r
+# install.packages("devtools")
+devtools::check()
+```
+
+### Style
+
+Match the existing code style. This means you should follow the tidyverse
+style guide (see [Resources](#resources)).
+You can also use the [styler](https://CRAN.R-project.org/package=styler) package
+to find code which does not adhere to the style guide.
+
+``` r
+# install.packages("styler")
+styler::style_pkg()
```

Comment:
I think this is bad practice because you only want to restyle what you've touched - I think that mean using the add-in will be most appropriate for most people.